src/endpoints.js

```diff
@@ -23,31 +30,16 @@ export function courtServerEndpoint() {
   }.backend.aragon.org`
 }
 
-// The graph endpoints
-const GRAPH_API_BASE_HTTP_LOCAL = 'http://127.0.0.1:8000'
-const GRAPH_API_BASE_WS_LOCAL = 'ws://127.0.0.1:8001'
-
-const GRAPH_API_BASE_HTTP = 'https://api.thegraph.com'
-const GRAPH_API_BASE_WS = 'wss://api.thegraph.com'
-
-const GRAPH_API_PATH = '/subgraphs/name/aragon/aragon-court'
-
-function getAPIBase() {
-  return isLocalOrUnknownNetwork(CHAIN_ID)
-    ? [GRAPH_API_BASE_HTTP_LOCAL, GRAPH_API_BASE_WS_LOCAL]
-    : [GRAPH_API_BASE_HTTP, GRAPH_API_BASE_WS]
+export function graphEndpoints() {
+  const { nodes } = getNetworkConfig()
+  return [nodes.subgraph.http, nodes.subgraph.ws]
 }
 
-export function graphEndpoints() {
-  const [API_BASE_HTTP, API_BASE_WS] = getAPIBase()
-  const networkType = isLocalOrUnknownNetwork()
-    ? 'rpc'
-    : getNetworkType(CHAIN_ID)
+export const defaultEthNode =
+  getDefaultEthNode() || getNetworkConfig().nodes.defaultEth
 
-  const API_PATH =
-    networkType === 'main'
-      ? GRAPH_API_PATH
-      : `${GRAPH_API_PATH}-${SUBGRAPH_NAME || networkType}`
+export const defaultIpfsGateway = getIpfsGateway()
 
-  return [`${API_BASE_HTTP}${API_PATH}`, `${API_BASE_WS}${API_PATH}`]
-}
+export const defaultSubgraphHttpEndpoint = getSubgraphHttpEndpoint()
+
+export const defaultSubgraphWsEndpoint = getSubgraphWsEndpoint()
```

Comment:
Just sanity checking if we still need this :)

Reply:
Removed in https://github.com/aragon/court-dashboard/pull/391